### PR TITLE
chore (refs T22614): Move core lib Docu.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T22614

- Move core lib Docu.

### CODE REVIEW

**Review by spiess-demos — COMMENTED** [review]
Is this file any helpful?

**Review by salisdemos — APPROVED** [review]

**Review by spiess-demos — CHANGES_REQUESTED** [review]
Maybe you could add the [contents of the actual documentation from demosplan-js-utils](https://github.com/demos-europe/demosplan-js-utils/blob/main/lib/lib_documentation.md?plain=1) here, again, before removing it there?

**Comment by ahmad-demos** [comment]
> Maybe you could add the [contents of the actual documentation from demosplan-js-utils](https://github.com/demos-europe/demosplan-js-utils/blob/main/lib/lib_documentation.md?plain=1) here, again, before removing it there?

Done in https://github.com/demos-europe/demosplan-core/pull/673/commits/999f76f081c5fa41800583aa78d031321a63fda3

File: client/js/lib/core/lib_documentation.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 ## Scripts, their effects and where they are used
 
+| CharCount.js | count characters in input fields | public and planning area |
+| CheckableItem.js | toggle checkboxes of multiple items | public and planning area |
 | ExtendedVueEvents.js | adds custom dplan taste to vue events | ? |
+| FloodControlField.js | used to enable a honeypot field to deter bots | not in use (?) |
+| FormActions.js | various methods to interact with forms | public and planning area |
+| GlobalEventListener | Add Event listeners, that have been added & initialized in mounted |
+| HighlightHashLink.js | used to highlight a currently active hashlink | used in faqlist.js (public) |
+| NotificationStoreAdapter.js | used to display flash messages | puplic and planning area |
```
Comment (spiess-demos) [inline client/js/lib/core/lib_documentation.md:10]: Can be removed here, as it has moved to to store.

```suggestion
```

File: client/js/lib/core/lib_documentation.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+| Pager.js | used to support a twig macro pager | public and planning area |
+| Progress.js | used to display a progress bar | Map.vue |
+| Tabs.js | used to add tabbing behaviour | public area |
```
Comment (spiess-demos) [inline client/js/lib/core/lib_documentation.md:13]: Can be removed here, as the code has moved into other locations.
```suggestion
```

File: client/js/lib/core/lib_documentation.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+| ToggleAnything.js | Adds toggle behavior to elements |
+| ToggleSideMenu.js | in/out. Saves state in sessionStorage |
```
Comment (spiess-demos) [inline client/js/lib/core/lib_documentation.md:15]:
```suggestion
| ToggleSideMenu.js | in/out. Saves state in sessionStorage |
| Tooltips.js | Initializes tooltips on elements with `o-tooltip` Css class |
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/lib/core/lib_documentation.md (hunk 3 above)
Comment (ahmad-demos) [reply client/js/lib/core/lib_documentation.md:12]: Done in https://github.com/demos-europe/demosplan-core/pull/673/commits/4e39a3da124fcef91a4fc6e2098ceb7b6cc17a39

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**